projects/laji-ui/src/lib/modal/modal/modal.component.ts

```diff
@@ -82,10 +82,8 @@ export class ModalComponent implements OnDestroy {
     if (this.isShown) {
       return;
     }
-    this.originalBodyOverflow = this.document.body.style.overflow;
     this.renderer.setStyle(this.elementRef.nativeElement, 'display', 'block');
```

Comment:
For consistency this could be refactored to function `showElement` (as there is the function `hideElement`)